app/controllers/articles_controller.rb

```diff
@@ -213,11 +209,8 @@ def handle_user_or_organization_feed
   end
 
   def handle_tag_feed
-    tag = Tag.find_by(name: params[:tag].downcase)
-
-    return unless tag
+    return unless (@tag = Tag.aliased_name(params[:tag]))
```

Comment:
I'm not sure saving one line here is fruitful for readability, I'd split this into:

```ruby
@tag = ...
return unless @tag
```